buffer/src/test/java/io/netty/buffer/AbstractCompositeByteBufTest.java

```diff
@@ -547,11 +662,15 @@ public void testReferenceCounts3() {
 
         c3.release(2);
         c2.release();
+
+        buf.removeComponent(2);
```

Comment:
it shouldn't be necessary to remove these components before hand ... all buffers should all have a reference count of 1 and the composite release should release them all right?